spec/clusterable/ht_item_spec.rb

```diff
@@ -173,4 +173,32 @@
       expect(no_ocn1.batch_with?(no_ocn3)).to be false
     end
   end
+
+  describe "Factory (spec/factories/ht_item.rb)" do
```

Comment:
These tests show that you can now, in tests using the factory for ht_items, set `billing_entity` directly and get the expected results.